### DISCUSSION TITLE
Best tool to auto document Tasks and properties (BuildParameters)

### DISCUSSION THREAD

**Discussion by nick5454** [cake-build/cake#3614] [General]
What is a good tool that I can use to auto-update a ReadMe file with comments. I've extended Cake.recipe file and I need a way so developers can look at the parameters to call with updating the readme file every time a change is made.

**Comment by pascalberger** [comment]
[Cake.Graph addin](https://github.com/cake-contrib/Cake.Graph) might be of interest for you
